src/algorithms/ForwardBackward.jl

```diff
@@ -139,7 +139,8 @@ function initialize!(sol::FBSIterator{I, R, D, CS, FS, AS, CQ, FQ, AQ, G}) where
         # 1) if adaptive = false and only fq is present then L is "accurate"
         # 2) otherwise L is "inaccurate" and set adaptive = true
         # TODO: implement case 1), now 2) is always performed
-        xeps = sol.x .+ sqrt(eps())
+        # xeps = sol.x .+ sqrt(eps())
+        xeps = (x -> x .+ sqrt(eps())).(sol.x)
```

Comment:
BTW this way of initializing L is not good, see PANOC.jl

Reply:
let's open an issue and solve this later!